workspace/notebook/nsip_s51_advice_migration.json

```diff
@@ -0,0 +1,182 @@
+				"auth": {
+					"type": "AAD",
+					"authResource": "https://dev.azuresynapse.net"
+				},
+				"sparkVersion": "3.3",
+				"nodeCount": 3,
+				"cores": 4,
+				"memory": 28,
+				"automaticScaleJobs": false
+			},
+			"sessionKeepAliveTimeout": 30
+		},
+		"cells": [
+			{
+				"cell_type": "markdown",
+				"metadata": {
+					"nteract": {
+						"transient": {
+							"deleting": false
+						}
+					}
+				},
+				"source": [
+					"## View odw_curated_db.vw_s51_advice is created"
+				]
+			},
+			{
+				"cell_type": "code",
+				"metadata": {
+					"jupyter": {
+						"source_hidden": false,
+						"outputs_hidden": false
+					},
+					"nteract": {
+						"transient": {
+							"deleting": false
+						}
+					},
+					"microsoft": {
+						"language": "sparksql"
+					},
+					"collapsed": false
+				},
+				"source": [
+					"\n",
+					"%%sql\n",
+					"\n",
+					"CREATE OR REPLACE VIEW odw_curated_migration_db.vw_s51_advice\n",
+					"    AS\n",
+					"SELECT DISTINCT\n",
+					"    AD.adviceId,\n",
+					"    AD.adviceReference ,\n",
+					"    CASE\n",
+					"        WHEN AD.caseId = 'None'\n",
+					"        THEN CAST(\"-1\" as INT)\n",
+					"        ELSE AD.caseId\n",
+					"    END                                                     AS caseId,\n",
+					"    AD.caseReference,\n",
+					"    AD.title,\n",
+					"    AD.titleWelsh,\n",
+					"    AD.from,\n",
+					"    CASE\n",
+					"        WHEN LOWER(AD.agent) = 'none'\n",
+					"        THEN NULL\n",
+					"        ELSE AD.agent\n",
+					"    END                                                     AS agent,\n",
+					"    CASE \n",
+					"        WHEN LOWER(AD.method) = 'none'\n",
+					"        THEN NULL\n",
+					"        ELSE LOWER(AD.method)\n",
+					"        END                                                 AS method,\n",
+					"    AD.enquiryDate,\n",
+					"    AD.enquiryDetails,\n",
+					"    AD.enquiryDetailsWelsh,\n",
+					"    AD.adviceGivenBy,\n",
+					"    AD.adviceDate,\n",
+					"    AD.adviceDetails,\n",
+					"    AD.adviceDetailsWelsh,\n",
+					"    CASE\n",
+					"        WHEN AD.status IN ('Not Checked','unchecked','Depublished')\n",
+					"        THEN \"unchecked\"\n",
+					"        WHEN AD.status = 'Do Not Publish'\n",
+					"        THEN 'donotpublish'\n",
+					"        ELSE LOWER(AD.status)\n",
+					"    END                                                     AS status,\n",
+					"    AD.redactionStatus,\n",
+					"    AD.attachmentIds,\n",
+					"    AD.ODTSourceSystem                                      AS SourceSystem\n",
+					"\n",
+					"FROM odw_harmonised_db.nsip_s51_advice AS AD\n",
+					"INNER JOIN (\n",
+					"    SELECT adviceId, MAX(IngestionDate) AS latest_date\n",
+					"    FROM odw_harmonised_db.nsip_s51_advice\n",
+					"    WHERE LOWER(ODTSourceSystem) ='horizon'\n",
+					"    GROUP BY adviceId\n",
+					") latest ON AD.adviceId = latest.adviceId AND AD.IngestionDate = latest.latest_date;   \n",
+					" \n",
```

Comment:
Do we not need a where outside the join as well? 
`WHERE LOWER(AD.ODTSourceSystem) = 'horizon';`